provider/README.md

```diff
@@ -151,45 +132,21 @@ For the purposes of this specification, the intersection of two geographic datat
 
 Municipalities requiring MDS Provider API compliance should provide an unambiguous digital source for the municipality boundary. This boundary must be used when determining which data each `provider` API endpoint will include.
 
-The boundary should be defined as a polygon or collection of polygons. The file defining the boundary should be provided in Shapefile or GeoJSON format and hosted online at a published address that all providers and `provider` API consumers can access and download.
+The boundary should be defined as a polygon or collection of polygons. The file defining the boundary should be provided in Shapefile or GeoJSON format and hosted online at a published address that all providers and `provider` API consumers can access and download. The boundary description can be sent as a reference to an object via the Geography API, or a reference to a Geography via the Jurisdiction API.
```

Comment:
If Jurisdiction isn't going to go with 1.0.0, I recommend we leave it out of this description for now.

Reply:
Jurisdiction is scheduled for 1.1.0, so agree on removing.

Reply:
fixed